install.sh

```diff
@@ -922,7 +925,7 @@ echo "$(
   cat <<EOS
 ${tty_bold}Read the analytics documentation (and how to opt-out) here:
   ${tty_underline}https://docs.brew.sh/Analytics${tty_reset}
-No analytics data has been sent yet (or will be during this \`install\` run).
+No analytics data has been sent yet (nor will be during this ${tty_bold}install${tty_reset} run).
```

Comment:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;'Nor' goes with a preceding 'neither,' as in 'neither …, nor ….'  Alternatively, give the dependent clause its own internal subject to avoid having to use that kind of construction:  
```suggestion
No analytics data has been sent yet (nor will it be during this ${tty_bold}install${tty_reset} run).
```

or

```suggestion
No analytics data has been sent yet (nor will any be during this ${tty_bold}install${tty_reset} run).
```